source/util.cpp

```diff
@@ -255,7 +256,7 @@ string generate_comment_for_declaration(clang::NamedDecl const *decl)
 // extract text from hierarchy of comments
 string get_text(comments::Comment const *C, SourceManager const & SM, SourceLocation previous)
 {
-	if( auto tc = dyn_cast<comments::TextComment>(C) ) return tc->getText();
+	if( auto tc = dyn_cast<comments::TextComment>(C) ) return string(tc->getText());
```

Comment:
also, i am a bit surprised by this change: Could you please double check that this conversion to string is indeed needed? (it should have happen automatically)

Reply:
Hi @lyskov ,

I was surprised too.  But that was  on newest fedora (i.e. newest gcc 10.2.1  and LLVM  11.0.0), so maybe they are stricter.

It was here:
https://github.com/andriish/binder/runs/1026649950   <- Fedora33. Are you able to see these runs?

```
/github/workspace/source/util.cpp: In function ‘std::string binder::get_text(const clang::comments::Comment*, const clang::SourceManager&, clang::SourceLocation)’:
/github/workspace/source/util.cpp:258:71: error: could not convert ‘tc->clang::comments::TextComment::getText()’ from ‘llvm::StringRef’ to ‘std::string’ {aka ‘std::__cxx11::basic_string<char>’}
  258 |  if( auto tc = dyn_cast<comments::TextComment>(C) ) return tc->getText();
      |                                                            ~~~~~~~~~~~^~
      |                                                                       |
      |                                                                       llvm::StringRef
```


Best regards,

Andrii

Reply:
got it, - thank you!